cEP-0010.md

```diff
@@ -0,0 +1,142 @@
+
+### Use letters instead of numbers for selecting actions
+
+Easier to remember an action based on the letters instead of numbers. (Do (n)othing).
+
+### Allow chaining actions in a single input
+
+Users could apply more than one patch in a single input. The idea here is to offer more flexibility to our users. Instead of just 1 number, the user could enter more actions.
+
+### Add a 'select action for all results' option
+
+Now users have the possibility to choose at every result an action. We want to add the possibility to choose an action for all results.
+
+### Reprompt after running actions
+
+We should have an option that lets user choose whether they want another action after the current action is done. This way, users will have the flexibility in their hands.
+
+### Set 'Do nothing' as the default action
+
+If a user chooses a non-existent action (or the input is wrong) just exit from the program with 'Do nothing'. This is for reuse of 'Do nothing'.
+
+The section below describe how we should make these changes.
+
+How
+---
+
+### Remove most of the text from the action selection screen
+
+Most of the text that should be changed is in the 'ConsoleInteraction.py' file. We should change that based on the feedback requested with
+Google Form.
+
+### Use letters instead of numbers for selecting actions
+
+Usage of letters would make usage of coala faster, as one could just remember
+that `a` stands for 'apply patch' for instance, and they would not have to check
+the list for every result to see what number 'apply patch' has been assigned.
+Masive changes should take place in 'result_actions' directory where are kept all the actions.
+
+### Allow chaining actions in a single input
+
+Mockup:
+
+[    ] Do (N)othing
+[    ] (O)pen file
+[    ] (A)pply the patch
+[    ] Add (i)gnore comment
+[    ] (C)hain actions
+[    ] Choose an action:
+
+If an user will press 'C':
+
+[    ] Enter the letters of the patches you want to apply('-' between any letter): a-i-n
+
+Any combination of letters (upper/lower) will be accepted (e.g: a-I-N).
+If one of the action requested isn't possible, an warning will be printed (after all possible actions are applied) with the letter that couldn't be satisfied.
+
+### Add a 'select action for all results' option
+
+Mockup:
+
+[    ] Do (N)othing
+[    ] (O)pen file(s)
+[    ] (A)pply the patch
+[    ] Add (i)gnore comment
+[    ] (C)hain actions
+[    ] (S)elect action for all results
+[    ] Choose an action:
+
+If the user will press 'S':
+
+==== Apply an action for all results ====
+[    ] Do (N)othing
+[    ] (O)pen file
+[    ] (A)pply the patch
+[    ] Add (I)gnore comment
+[    ] Choose an action:
+
+So, let's say that a user wants to add an ignore comment in all files.
+The user should choose '(S)elect action for all results' and after, 'i' option.
```

Comment:
I think "and Choose the 'i' option" would fit better. The comma placement is confusing.